src/index.html

```diff
@@ -15,8 +15,100 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo__img"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="list__item">
+            <a
+              class="nav__link is-active"
+              href="index.html"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              class="nav__link"
+              href="#"
```

Comment:
The `href` attribute is set to `#`, which is typically a placeholder. Ensure this is updated to the correct URL for the 'Tablets' link.